portfolio/src/main/webapp/script.js

```diff
@@ -11,18 +13,60 @@
 // WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 // See the License for the specific language governing permissions and
 // limitations under the License.
+ 
+/**
+ * Gives the user a random book recommendation.
+ */
+
+ 
+function sayRandomBook(){
+    const books = [
+        "Jane Eyre", "1984","Don Quixote", "The Great Gatsby", "Harry Potter and the Philosopher's Stone",
+        "War and Peace", "The Alchemist", "The Lord of the Rings", "Things fall apart", "Lord of the Flies", "Animal Farm",
+        "The Lion, The Witch, and the Wardrobe","Percy Jackson and the Lightning Thief", "Honor among thieves", "Summer Nights at the Midnight Hotel", "It", "The Hound of the Baskervilles",
+    ];
+ 
+    const chosenbook= books[Math.floor(Math.random() * books.length)];
```

Comment:
The google style guide says to use lowerCamelCase for variable names.

https://google.github.io/styleguide/jsguide.html#naming-local-variable-names